src/cbor/common.h

```diff
@@ -285,6 +287,16 @@ size_t cbor_refcount(const cbor_item_t *item);
  */
 cbor_item_t *cbor_move(cbor_item_t *item);
 
+/** Compares two items
+ *
+ * Compares the type, metadata and value of item1 and item2
```

Comment:
This definitely requires more docs, please see the top level comment

Reply:
doc improved, let me know if more information would be useful